.env.example

```diff
@@ -0,0 +1 @@
+PORT=4000
```

Comment:
Not sure but isn't this file name supposed to be `.env` not `.env.example`? :thinking: 
It didn't work for me until I renamed it.

Reply:
I just now realized, that was the point .. :sweat_smile:

Reply:
Whoops, sorry about that. I actually meant to copy the respective line from the termbox README for that. Done now!